### PR TITLE
Raphael Fritz Woche 3 Aufgaben

### PR DESCRIPTION
**Pull request by FritzRa**

### CODE REVIEW

File: Exercises/Raphael Fritz/Woche3.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        for suits in ["diamonds", "hearts", "spades", "clubs"]:
+            for type in ["2","3","4","5","6","7","8","9","10","Jack","Queen","King"]:
+                deck.append(type+" of "+suits)
+        super().__init__(deck)
+                
+class SkatDeck(DeckOfCards):
+    def __init__(self):
+        deck = []
+        for suits in ["diamonds", "hearts", "spades", "clubs"]:
+            for type in ["7","8","9","10","Jack","Queen","King"]:
```
Comment (GeFritz) [inline Exercises/Raphael Fritz/Woche3.py:42]: ace is missing

Comment (FritzRa) [reply Exercises/Raphael Fritz/Woche3.py:42]: on it!

File: Exercises/Raphael Fritz/Woche3.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# PART 2:
+# Create a second class that represents a deck of cards usable for Skat -- it should only contain cards from 7 upwards.
+# It should offer all the same functionality of the first class.
+
+
+# Write some code to test the functionality of both kinds of decks. (You can use `assert` to make sure your classes behave the way you expect them to.)
+class DeckOfCards:
+    def __init__(self,deck):
+        self.deck = deck
```
Comment (RubelOS) [inline Exercises/Raphael Fritz/Woche3.py:19]: This is a nice relation. Good idea!

File: Exercises/Raphael Fritz/Woche3.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        return str(", ".join(self.deck))
+    def __getitem__(self,index):
+        if 0 <= index < len(self.deck):
+            return self.deck[index]
+        else:
+            raise IndexError("Index out of range")
+
+class FrenchDeck(DeckOfCards):
+    def __init__(self):
+        deck = []
```
Comment (RubelOS) [inline Exercises/Raphael Fritz/Woche3.py:32]: what would u even need the `DeckOfCards` for if you just use a new list

File: Exercises/Raphael Fritz/Woche3.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+#
+# run your function with the lower bound `134564` and the upper bound `585159`. Note the resulting count
+# in your pull request, please.
+
+
+
+def analyser(num):
+    increase=True
+    grouping=False
+    for index in range(len(num)-1):
```
Comment (RubelOS) [inline Exercises/Raphael Fritz/Woche3.py:80]: solved nicely

**Review by RubelOS — APPROVED** [review]
All in all, a nice solution to the task. Awesome!👌

**Pull request merged**